lib/active_fedora/file_persistence.rb

```diff
@@ -0,0 +1,27 @@
+module ActiveFedora
+  module FilePersistence
+    extend ActiveSupport::Concern
+
+    include ActiveFedora::Persistence
+
+    private
+
+      def _create_record(_options = {})
+        return false if content.nil?
+        ldp_source.content = content
+        ldp_source.create do |req|
+          req.headers.merge!(ldp_headers)
+        end
+        refresh
+      end
+
+      def _update_record(_options = {})
+        return false unless content_changed?
```

Comment:
I agree, it should return true if there is nothing to save as ActiveRecord does. See https://gist.github.com/jcoyne/074969310111653ae02c